assets/src/edit-story/elements/media/util.js

```diff
@@ -78,24 +78,22 @@ export function calculateSrcSet(resource) {
 }
 
 /**
- * Choose the source URL of the smallest available size image wider than
+ * Choose the source URL of the smallest available size image / video wider than
  * minWidth, according to the device pixel ratio.
  *
  * @param {number} minWidth The minimum width of the thumbnail to return.
- * @param {*} resource Image resource object.
- * @return {string} Source URL of the smallest available size image.
+ * @param {*} resource Resource object.
+ * @return {string} Source URL of the smallest available size media.
  */
-function getThumbnailUrl(minWidth, resource) {
+export function getThumbnailUrl(minWidth, resource) {
```

Comment:
It might be a good moment to rename this method to something better, as it doesn't necessarily return a thumbnail anymore, just the appropriate size. Maybe getResourceSizeForWidth?